css/buttons.css

```diff
@@ -0,0 +1,90 @@
+/* GENERAL */
```

Comment:
as it also contains general styling I would call it `main.css` and not `buttons.css`
look & normalize your indentation (it makes things easier to read)